soccer/ui/MainWindow.ui

```diff
@@ -82,6 +82,9 @@
              <height>24</height>
```

Comment:
Hmm Thats weird. It should really only jiggle from changes in number of digits now.